File: 01. Iniciante/C/URI1008.c
// URI 1008. Salário

#include <stdio.h>

int main()
{
    int idFunc, hrTrab;
    float vlrHora, salario;

    scanf("%d%d%f", &idFunc, &hrTrab, &vlrHora);

    salario = vlrHora * hrTrab;

    printf("NUMBER = %d\n", idFunc);
    printf("SALARY = U$ %.2f\n", salario);

    return 0;
}

File: 01. Iniciante/Java/URI1008.java
import java.io.IOException;
import java.util.Scanner;

// Alterar a classe para Main no URI
public class URI1008 {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        Integer idFunc = sc.nextInt();
        Integer hrsTrab = sc.nextInt();
        Double vlrHora = sc.nextDouble();
        Double salario = hrsTrab * vlrHora;
        System.out.printf("NUMBER = %d\nSALARY = U$ %.2f\n", idFunc, salario);
        sc.close();
    }
}
